aptos-move/framework/supra-framework/sources/configs/supra_config.move

```diff
@@ -0,0 +1,70 @@
+/// Maintains the consensus config for the blockchain. The config is stored in a
+/// Reconfiguration, and may be updated by root.
+module supra_framework::supra_config {
+    use std::error;
+    use std::vector;
+    use supra_framework::chain_status;
+    use supra_framework::config_buffer;
+
+    use supra_framework::reconfiguration;
+    use supra_framework::system_addresses;
+
+    friend supra_framework::genesis;
+    friend supra_framework::reconfiguration_with_dkg;
+
+    struct SupraConfig has drop, key, store {
+        config: vector<u8>,
+    }
+
+    /// The provided on chain config bytes are empty or invalid
+    const EINVALID_CONFIG: u64 = 1;
+
+    /// Publishes the SupraConfig config.
+    public(friend) fun initialize(supra_framework: &signer, config: vector<u8>) {
+        system_addresses::assert_supra_framework(supra_framework);
+        assert!(vector::length(&config) > 0, error::invalid_argument(EINVALID_CONFIG));
+        move_to(supra_framework, SupraConfig { config });
+    }
+
+    /// Deprecated by `set_for_next_epoch()`.
+    ///
+    /// WARNING: calling this while randomness is enabled will trigger a new epoch without randomness!
+    ///
+    /// TODO: update all the tests that reference this function, then disable this function.
+    public fun set(account: &signer, config: vector<u8>) acquires SupraConfig {
+        system_addresses::assert_supra_framework(account);
+        chain_status::assert_genesis();
+        assert!(vector::length(&config) > 0, error::invalid_argument(EINVALID_CONFIG));
+
+        let config_ref = &mut borrow_global_mut<SupraConfig>(@supra_framework).config;
+        *config_ref = config;
+
+        // Need to trigger reconfiguration so validator nodes can sync on the updated configs.
+        reconfiguration::reconfigure();
+    }
```

Comment:
If this is a deprecated method in the original one, there is no need for us to have this now. Better to remove this and test.

Reply:
Thanks for pointing this out. I've removed the method.